packages/plugins/learn-card-network/src/types.ts

```diff
@@ -37,6 +37,7 @@ import { VerifyExtension } from '@learncard/vc-plugin';
 
 /** @group LearnCardNetwork Plugin */
 export type LearnCardNetworkPluginDependentMethods = {
+    __init__?: () => Promise<void>;
```

Comment:
Should we add some documentation/comment around using this in scripts/tests

Reply:
I actually didn't end up taking this approach! LearnCloud plugin was able to do it all behind the scenes, and the LCN plugin can just have you await `getProfile` (which is what the test does!)

I'll add some docs for that though =)